.github/workflows/example-chrome.yml

```diff
@@ -14,39 +14,40 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Cypress info
+      - name: Chrome headless
         uses: ./
         with:
-          # just for full picture after installing Cypress
-          # print information about detected browsers, etc
+          # Print information about the system and current environment
+          # including detected browsers
+          # which are pre-installed in GitHub-hosted runners
           # see https://on.cypress.io/command-line#cypress-info
+          # For convenience, (mis-)use build parameter to get cypress info
```

Comment:
Can you explain this one in more detail? I didn't understand the comment with "(mis-)use"

Reply:
I will take out the `mis-` part. You are right to comment because I didn't explain it.

`build` is a parameter to build an app, for instance using `npm run build`. In this case we are using the parameter for something completely different, to display configuration information via `npx cypress info`.

Reply:
comments re-worded in https://github.com/cypress-io/github-action/pull/1035/commits/2aa98eb6d1cf2a1c883a31d9c8362e55be1ad010